### PR TITLE
MHP-3002: Suggest landscape

### PR DESCRIPTION
**Pull request by OzzieOrca**

### CODE REVIEW

File: app/assets/javascripts/angular/components/myPeopleDashboard/myPeopleDashboard.service.js [hunk 1]
```diff
@@ -4,6 +4,11 @@ angular
 
 function myPeopleDashboardService(httpProxy, modelsService, _) {
     var myPeopleDashboardService = {
+        isMobile: () =>
+            window.navigator.userAgent.match(
+                /Android|webOS|iPhone|iPad|iPod|BlackBerry|Windows Phone/i,
```
Comment (OzzieOrca) [inline app/assets/javascripts/angular/components/myPeopleDashboard/myPeopleDashboard.service.js]: It's back from the dead haha 💀

Comment (TheNoodleMoose) [reply app/assets/javascripts/angular/components/myPeopleDashboard/myPeopleDashboard.service.js]: Oh no, Halloween came early 😆

**Comment by TheNoodleMoose** [comment]
Should we add the test back in as well? 😆

**Comment by TheNoodleMoose** [comment]
Oh yeah that's true. Yeah its not a very complex test so I think we'll be okay without it haha.

**Pull request merged**